data-source/json/test_radio_open.json

```diff
@@ -0,0 +1,128 @@
+{
+	"mime_type": "application/json/ons/eq",
+	"schema_version": "0.0.1",
+	"data_version": "0.0.3",
+	"survey_id": "0",
+	"title": "Radio Open",
+	"theme": "default",
+	"metadata": [{
+			"name": "user_id",
+			"type": "string"
+		},
+		{
+			"name": "period_id",
+			"type": "string"
+		},
+		{
+			"name": "ru_name",
+			"type": "string"
+		}
+	],
+	"sections": [{
+		"id": "section-1",
+		"groups": [{
+			"id": "radio",
+			"title": "Radio Open",
+			"blocks": [{
+					"type": "Question",
+					"id": "radio-open-true",
+					"question": {
+						"type": "General",
+						"id": "radio-open-true-question",
+						"title": "What is your favourite drink?",
+						"answers": [{
+							"type": "Radio",
+							"id": "radio-open-true-answer",
+							"mandatory": false,
+							"voluntary": true,
+							"options": [{
+									"label": "Coffee",
+									"value": "Coffee"
+								},
+								{
+									"description": "Select to enter answer",
```

Comment:
The description "Select to enter answer" should be removed as it's misleading when the detail answer is always open.

Reply:
I removed it from test schema. There are still two census questions affected, but I had to ask someone for exact wording on these descriptions and waiting for reply now.

Reply:
Ok, descriptions have been removed from census questions, test schema description has been refactored. Thanks!